securedrop/config.py.example

```diff
@@ -83,3 +83,8 @@ TEMP_DIR = os.path.join(SECUREDROP_DATA_ROOT, "tmp")
 # depending on the environment
 DATABASE_ENGINE = 'sqlite'
 DATABASE_FILE = os.path.join(SECUREDROP_DATA_ROOT, 'db.sqlite')
+
+# Which of the available locales should be displayed by default ?
+# To edit, modify the value of securedrop_default_locale
+# in install_files/ansible-base/roles/app/defaults/main.yml
+DEFAULT_LOCALE = 'en_US'
```

Comment:
The comment about `install_files/ansible-base/roles/development/tasks/main.yml` is not inaccurate, but in practice we will likely add the `securedrop_default_locale` var to the `./securedrop-admin sdconfig` flow, which will write it to `install_files/ansible-base/group_vars/all/site-specific`. Docs for that won't land until we have full i18n, however, so no changes necessary at this time.

Reply:
Ah, that makes sense, thanks for the detailed explanation. I removed the **To edit ...** part.